### ISSUE TITLE
Simple Kafka Consumer

### ISSUE THREAD

**Issue by danielwegener** [softwaremill/reactive-kafka#5]
This request somewhat relates to #3. It would be very useful if one could materialize a partition source with a given offset - leaving the offset management completely to the application. 

possible api:
```scala
class ReactiveKafka(val host: String, val zooKeeperHost: String) {
/** consumes from a given topic/partition at a given offset and provides a stream of tuples of offset and message */ 
def consume(topic: String, partitionId: Int, startOffset:Long)(implicit actorSystem: ActorSystem): Publisher[(Long,String)] = { ...
```

Though, it might be a bit of work to create a robust simple consumer ([example from the kafka-wiki(]https://cwiki.apache.org/confluence/display/KAFKA/0.8.0+SimpleConsumer+Example)).

**Comment by mjkrumlauf** [comment]
This can be closed - #63 took care of this.

**Comment by akauppi** [comment]
>This can be closed - #63 took care of this.

Just noticed this while browsing the issues...
